.github/workflows/build_and_run_example_test.yml

```
@@ -0,0 +1,96 @@
+name: Build Websockets
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        idf_ver: ["latest"]
+        idf_target: ["esp32"]
+
+    runs-on: ubuntu-20.04
+    container: espressif/idf:${{ matrix.idf_ver }}
+    steps:
+      - name: Checkout esp-protocols
+        uses: actions/checkout@v1
+        #with:
+        #  path: esp-protocols
+      - name: Build ${{ matrix.example }} with IDF-${{ matrix.idf_ver }} for ${{ matrix.idf_target }}
+        env:
+          IDF_TARGET: ${{ matrix.idf_target }}
+        shell: bash
+        working-directory: components/esp_websocket_client/examples/
+        run: |
+          . ${IDF_PATH}/export.sh
+          cp sdkconfig.ci sdkconfig.defaults
```

This replaces the `sdkconfig.defaults`, while in IDF CI we define additional options.
I'd suggest appending the `sdkconfig.ci` options to the default one, for example:
```suggestion
          cat sdkconfig.ci >> sdkconfig.defaults
```